module_lambdacron_cache_sync.tf

```diff
@@ -0,0 +1,51 @@
+module "lambdacron_cache_sync" {
+  source = "./modules/lambdacron"
+
+  project        = var.project
+  environment    = var.environment
+  component      = var.component
+  aws_account_id = var.aws_account_id
+  region         = var.region
+  aws_partition  = var.aws_partition
+  aws_url_suffix = var.aws_url_suffix
+
+  function_name         = "commonfate-cache-sync"
```

Comment:
is commonfate-cache-sync right here?

Aligned to the other lambdas and the package names and such, I would expect this function name to just be "cache-sync"